### PR TITLE
cli template tokens

### PR DESCRIPTION
**Pull request by paul-krohn**
Allow passing of arbitrary replacement tokens. For example, `plumbum --token account_alias=main <namespace> <template>` will replace `{{ account_alias }}` in `<template>` with `main`.

### CODE REVIEW

File: plumbum.py [hunk 1]
```diff
@@ -200,11 +195,19 @@ def main():
...
         'resources': resources,
-    }))
+    }
+    # add tokens passed as cli args:
+    if tokens is not None:
+        for token_pair in tokens:
+            (key, value) = token_pair.split('=')
+            template_tokens[key] = value
+
+    print(template.render( template_tokens))
```
Comment (jib) [inline plumbum.py]: nit: whitespace

File: plumbum.py [hunk 2]
```diff
@@ -200,11 +195,19 @@ def main():
...
+    template_tokens = {
         'filters': filters,
         'region': region,       # Use for Auth config section if needed
         'resources': resources,
-    }))
+    }
+    # add tokens passed as cli args:
+    if tokens is not None:
+        for token_pair in tokens:
+            (key, value) = token_pair.split('=')
```
Comment (jib) [inline plumbum.py:212]: does this need an input check, to make sure the user actually passed foo=bar tokens?

Comment (paul-krohn) [inline plumbum.py:212]: Good idea, added.

File: plumbum.py [hunk 3]
```diff
@@ -81,24 +84,19 @@ def lookup(instances, filter_by=None):
 
 def interpret_options(args=sys.argv[1:]):
 
-    if '--version' in args:
-        print(__version__)
-        sys.exit()
-
     parser = argparse.ArgumentParser()
+    parser.add_argument('--version', action='version', version=__version__)
```
Comment (hanpeter) [inline plumbum.py:88]: Oh, there is a built-in `version` option handling? Awesome.

File: plumbum.py [hunk 4]
```diff
@@ -200,11 +198,21 @@ def main():
...
-        'region': region,       # Use for Auth config section if needed
+        'region': region,  # Use for Auth config section if needed
         'resources': resources,
-    }))
+    }
+    # add tokens passed as cli args:
+    if tokens is not None:
+        for token_pair in tokens:
+            if token_pair.count('=') != 1:
+                raise CliArgsException("token pair '{}' invalid, must contain exactly one '=' character.".format(token_pair))
```
Comment (hanpeter) [inline plumbum.py]: WARNING: `{}` in the format string is only supported in Python 2.7 / trusty.

Comment (paul-krohn) [inline plumbum.py]: :angry: thanks for the heads-up!

**Comment by hanpeter** [comment]
LGTM

**Pull request merged**